test/fixtures/plugin.filler/fill-radar-dataset-spline.json

```diff
@@ -27,6 +27,7 @@
         },
         "options": {
             "responsive": false,
+            "spanGaps": false,
```

Comment:
Does the default behavior change?
If so, shouldn't this be specified in dataset defaults to preserve old behavior and avoid breaking change?

Reply:
No, the default doesn't change. This is not required here, but It just makes it clear that `spanGaps` is disabled in this test.